guide/guide-qip.rst

```diff
@@ -53,7 +80,7 @@ Gate name                           Description
 "CRY"                 Controlled rotation around y axis
 "CRZ"                 Controlled rotation around z axis
 "CPHASE"              Controlled phase gate
-"CNOT"                Controlled NOT gate 
+"CNOT"                Controlled NOT gate
```

Comment:
We now also have X,Y,Z,T,S gates. Although they are not from your project, Could you also add them here in the list? Thanks!

Reply:
Sure.